### PR TITLE
Add download link and fix naming

### LINKED ISSUE #8: Publish a Release
We should publish a release of SmartDashboard here on github.

### PR DESCRIPTION
**Pull request by AustinShalit**
closes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
 
 ## WPILib Mission
 
 The WPILib Mission is to enable FIRST teams to focus on writing game-specific software rather than on hardware details - "raise the floor, don't lower the ceiling". We try to enable teams with limited programming knowledge and/or mentor experience to do as much as possible, while not hampering the abilities of teams with more advanced programming capabilities. We support Kit of Parts control system components directly in the library. We also strive to keep parity between major features of each language (Java, C++, and NI's LabVIEW), so that teams aren't at a disadvantage for choosing a specific programming language. WPILib is an open-source project, licensed under the BSD 3-clause license. You can find a copy of the license [here](BSD_License_for_WPILib_code.txt).
 
-# Building Smart Dashboard
+# Getting SmartDashboard
 
-Building Smart Dashboard is very straightforward. Smart Dashboard uses Gradle to compile.
+The latest release build of SmartDashboard can be found on [WPI's server](http://first.wpi.edu/FRC/roborio/maven/release/edu/wpi/first/wpilib/SmartDashboard/).
```
Comment (333fred) [inline README.md]: Perhaps a bit more detail, since the file system viewer that WPI uses doesn't show file extensions

Comment (AustinShalit) [inline README.md]: Maybe we should fix that... @nightpool

Comment (333fred) [inline README.md]: Not fixable by us, unfortunately. That's an official wpi server.

Comment (AustinShalit) [inline README.md]: Do you think the best option is to document that the largest file is the jar?

Comment (333fred) [inline README.md]: And mention that hovering on the link shows the full path, including the extension.

Comment (AustinShalit) [inline README.md]: The hovering trick does not work for me... 😢

Comment (nightpool) [inline README.md]: @333fred @AustinShalit I can work with brad to get this fixed if we know how we want to host stuff. Is this just Jenkins artifacts?

Comment (333fred) [inline README.md]: @AustinShalit, your browse doesn't show full links at the bottom right? What are you running? And @nightpool, the idea with first.wpi.edu is that we _don't_ want to host stuff.

Comment (AustinShalit) [inline README.md]: I am running Safari.

Comment (Twinters007) [inline README.md]: @AustinShalit `View -> Show Status Bar`

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ The WPILib Mission is to enable FIRST teams to focus on writing game-specific so
 
 # Getting SmartDashboard
 
-The latest release build of SmartDashboard can be found on [WPI's server](http://first.wpi.edu/FRC/roborio/maven/release/edu/wpi/first/wpilib/SmartDashboard/).
+The latest release build of SmartDashboard can be found on [WPI's server](http://first.wpi.edu/FRC/roborio/maven/release/edu/wpi/first/wpilib/SmartDashboard/).  The largest file listed is the jar file.  You can check by hovering over the link which will reveal the full path, including the extension.
```
Comment (Twinters007) [inline README.md]: When you click on this link, you still have to go into the version folder. You should add this step in (or edit the server to have a `latest` folder and change the link to this folder)

File: README.md [hunk 3]
```diff
@@ -1,29 +1,35 @@
...
+
+Building SmartDashboard is very straightforward. SmartDashboard uses Gradle to compile.
 
 ## Requirements
 - [JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
 
 ## Running
 
-To run Smart Dashboard navigate to the `smartdashboard` directory and use the command `./gradlew :run`.
+To run SmartDashboard navigate to the `smartdashboard` directory and use the command `./gradlew :run`.
```
Comment (Twinters007) [inline README.md]: `:run`?

Comment (AustinShalit) [inline README.md]: Yes:  `./gradlew :run`

Comment (AustinShalit) [inline README.md]: Just for laughs:

https://www.chiefdelphi.com/forums/showthread.php?t=133221

Comment (Twinters007) [inline README.md]: kek

**Pull request merged**